docs/content/reference/install-configuration/api-dashboard.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Traefik API & Dashboard Documentation"
+description: "Traefik Proxy exposes information through API handlers and showcase them on the Dashboard. Learn about the security, configuration, and endpoints of the APIs and Dashboard. Read the technical documentation."
+---
+
+The dashboard is the central place that shows you the current active routes handled by Traefik.
+
+<figure>
+    <img src="../../../assets/img/webui-dashboard.png" alt="Dashboard - Providers" />
+    <figcaption>The dashboard in action</figcaption>
+</figure>
+
+## Configuration Options
```

Comment:
To be consistent with other sections the example should be before the config options description.